### ISSUE TITLE
Wanted: internationalization support.

### ISSUE THREAD

**Issue by ZoneTwelve** [bclswl0827/ChatGemini#19]
If there are any opportunities for you or your team to implement internationalization, that would be great.

> When I have more free time, I'd like to chat and see how I can help.

**Comment by bclswl0827** [comment]
Of course, this is a personal project, and your contributions are highly appreciated :)
